tests/test_http_client.py

```diff
@@ -51,6 +52,25 @@ def test_new_default_http_client_urllib2(self, request_mocks):
         )
 
 
+class TestNewDefaultHttpClientAsync(StripeClientTestCase):
+    def check_default(self, none_libs, expected):
+        for lib in none_libs:
+            setattr(_http_client, lib, None)
+
+        inst = _http_client.new_default_http_client_async()
+
+        assert isinstance(inst, expected)
+
+    def test_new_default_http_client_httpx(self, request_mocks):
+        self.check_default((), _http_client.HTTPXClient)
+
+    def test_new_default_http_client_no_import_found(self, request_mocks):
```

Comment:
Does this check other behaviors like timeouts and retries?

Reply:
No this is just a test for the behavior of the `new_default` function delegating to the correct constructors.

Reply:
In https://github.com/stripe/stripe-python/pull/1165/commits/cd9c8bb3998c4eb5b0239a1461c9f7c9348b9005 I added more tests to mirror the ones elsewhere in `test_http_client`. This led me to add support for `proxies` and `verify` like Requests has, too.